buildutils/src/create-test-package.ts

```diff
@@ -0,0 +1,38 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import * as utils from './utils';
+
+
+if (require.main === module) {
+  // Make sure we have required command line arguments.
+  if (process.argv.length !== 3) {
+      let msg = '** Must supply a source package name\n';
+      process.stderr.write(msg);
+      process.exit(1);
+  }
+  let name = process.argv[2];
+  let pkgPath = path.resolve(path.join('.', 'packages', name));
+  if (!fs.existsSync(pkgPath)) {
+    console.error('Package does not exist: ', name);
+    process.exit(1);
+  }
+  let dest = path.resolve(`./tests/test-${name}`);
```

Comment:
should `path.join` be used here?

Reply:
Backslash paths are still valid in Windows.

Reply:
+1